sunpy/instr/goes.py

```diff
@@ -138,7 +230,7 @@ def temp_em(goeslc, abundances="coronal", download=False, download_dir=DATA_PATH
     2014-01-01 00:00:02  7e-07  7e-06
     2014-01-01 00:00:04  7e-07  7e-06
     2014-01-01 00:00:06  7e-07  7e-06
-    >>> goeslc_new = temp_em(goeslc)
+    >>> goeslc_new = calculate_temperature_em(goeslc)
```

Comment:
On the topic of `_new` while I agree it is a little odd, it does make the point that the object returned is not the same lightcurve as was passed in. I am not sure I have a better suggestion?